--- bonobo-conf-0.9/acinclude.m4~	Thu Dec 21 07:48:45 2000
+++ bonobo-conf-0.9/acinclude.m4	Thu Aug  2 11:44:52 2001
@@ -68,7 +68,7 @@
 ])
 dnl See whether we need a declaration for a function.
 dnl GCC_NEED_DECLARATION(FUNCTION [, EXTRA-HEADER-FILES])
-AC_DEFUN(GCC_NEED_DECLARATION,
+AC_DEFUN([GCC_NEED_DECLARATION],
 [AC_MSG_CHECKING([whether $1 must be declared])
 AC_CACHE_VAL(gcc_cv_decl_needed_$1,
 [AC_TRY_COMPILE([
@@ -101,7 +101,7 @@
 
 dnl Check multiple functions to see whether each needs a declaration.
 dnl GCC_NEED_DECLARATIONS(FUNCTION... [, EXTRA-HEADER-FILES])
-AC_DEFUN(GCC_NEED_DECLARATIONS,
+AC_DEFUN([GCC_NEED_DECLARATIONS],
 [for ac_func in $1
 do
 GCC_NEED_DECLARATION($ac_func, $2)
